donut/modules/calendar/templates/share_cal-iframe.html

```diff
@@ -0,0 +1,58 @@
+{% extends "layout.html" %}
+{% block navbar %}{% endblock %}
+{% block styles %}
+  {{super()}}
+  <link rel="stylesheet" href={{ url_for('calendar.static', filename='style.css')}}>
+{% endblock %}
+{% block content %}
+<base target="_parent" />
+<div class="col-md-8 col-md-offset-2">
+<form method="POST" action="{{ url_for('calendar.calendar_share_cal') }}">
+    <table id="calendar" class="table">
+      <tr>
+       <div class="form-group">
+          <td class="calendar">
+            <label for="email" class="required">Google Email:</label>
+          </td>
+          <td class="calendar"><input type="text" name="email" class="form-control" required /></td>
+        </div>
+      </tr>
+      <tr>
+        <div class="form-group">
+          <td class="calendar">
+            <label for="tag" class="required">Calendar:</label>
+          </td>
+          <td class="calendar">
+            {% for item in permissions %}
+              {% if item != 'Any' %}
+                <label class="checkbox-inline"><input type="checkbox" name="tag" value="{{ item }}">{{ item }}</label>
+              {% endif %}
+            {% endfor %}
+          </td>
+        </div>
+      </tr>
+      <tr>
+        <div class="form-group">
+          <td class="calendar">
+             <label for="access_level" class="required">Access Level (if you selected multiple and request edit-level, it will apply them automatically
+            ):</label>
+          </td>
+          <td class="calendar">
+            <select class="form-control" name="access_level">
+              <option value="reader">Read</option>
+              {% if permissions %}
+                <option value="writer">Edit</option>
+              {% endif %}
+            </select>
+          </td>
+        </div>
+      </tr>
+    </table>
```

Comment:
Is there a reason to put this in a `table`? I think Bootstrap forms are generally more readable when not in a table

Reply:
I think it looked better... I wanted it such that the inputs floats to the right of the text